react/src/components/molecules/MainNav/index.js

```diff
@@ -104,7 +105,7 @@ class MainNav extends Component {
               id: liId
             };
             return(
-              <li {...liClasses} onMouseEnter={(e) => this.mouseOver(e)} onMouseLeave={(e) => this.mouseOut(e)}>
+              <li {...liClasses} role="presentation" onMouseEnter={(e) => this.mouseOver(e)} onMouseLeave={(e) => this.mouseOut(e)}>
```

Comment:
bind function to this...
```suggestion
               <li {...liClasses} role="presentation" onMouseEnter={(e) => this.mouseOver(e)} onMouseLeave={(e) => this.mouseOut(e)} onKeyDown={(e) => this.onKeyDown(e)}>
```